lib/pure/redis.nim

```diff
@@ -798,6 +798,35 @@ proc zunionstore*(r: Redis, destination: string, numkeys: string,
   
   return r.readInteger()
 
+# HyperLogLog
+
+proc pfadd*(r: Redis, key: string, elements: varargs[string]): RedisInteger = 
+  ## Add variable number of elements into special 'HyperLogLog' set type
+  var args = @[key]
+  for element in items(elements):
+    args.add(element)
+
+  r.sendCommand("PFADD", args)
+
+  return r.readInteger()
+
+proc pfcount*(r: Redis, key: string): RedisInteger =
+  ## Count approximate number of elements in 'HyperLogLog'
+  var args = @[key]
+
+  r.sendCommand("PFCOUNT", args)
+
+  return r.readInteger()
+
+proc pfmerge*(r: Redis, destination: string, sources: varargs[string]) =
+  ## Merge several source HyperLogLog's into one specified by destKey
+  var args = @[destination]
+  for source in items(sources):
+    args.add(source)
+
+  r.sendCommand("PFMERGE", args)
```

Comment:
Create a version of 'sendCommand' that can take a single string? Saves some seq creations.